LICENSE

```diff
@@ -47,7 +47,7 @@ authors' sake, the GPL requires that modified versions be marked as
 changed, so that their problems will not be attributed erroneously to
 authors of previous versions.
 
-  Some devices are designed to deny users access to install or run
+  Some images are designed to deny users access to install or run
```

Comment:
this must be rolled back (we shouldn't tinker with the license). Perhaps this is the result of a "replace all"?